### ISSUE TITLE
Plugin doesn't seem to work at all if the Vault is located in a different network location

### ISSUE THREAD

**Issue by HonestyK** [trganda/obsidian-attachment-management#131]
I've tested this plugin on a Vault that's local (e.g My Documents) and a Vault that's in a folder on the local network, and I noticed the plugin works fine for the former but not the latter. Just wondering if this is a bug or is there some limitation with Obsidian that's responsible for this.

There's no error message given at all; it's as if it was never installed even though it's Enabled and everything.

**Comment by trganda** [comment]
Yeah, there is some limitation on the Obsidian. Some file operation API was not working expected on network location, e.g. SMB.
